Sources/Rendering/Core/AbstractImageMapper/index.d.ts

```diff
@@ -0,0 +1,58 @@
+import vtkAbstractMapper, { IAbstractMapperInitialValues } from "../AbstractMapper";
+import vtkImageData from "../../../Common/DataModel/ImageData";
+
+
+export interface IAbstractImageMapperInitialValues extends IAbstractMapperInitialValues {
+	slice?: number;
+}
+
+export interface vtkAbstractImageMapper extends vtkAbstractMapper {
+
+	/**
+	 *
+	 */
+	getIsOpaque(): boolean;
+
+	/**
+	 * @returns currently active image data object
+	 */
+	getCurrentImage(): vtkImageData | null;
+
+	/**
+	 * To be overridden by deriving types
+	 * @param slice return the image data object corresponding to a slice number.
+	 */
+	getImage(slice: number): vtkImageData | null;
```

Comment:
This function seems very specific to ImageArrayMapper. Do you really need it in such superclass ?

Reply:
This is suprising to me especially since ImageArrayMapper does not derive from ImageMapper.

Reply:
yes, the function is not needed in `AbstractImageMapper` or `ImageMapper`. I will remove it.